packages/sanity/src/core/releases/tool/components/ReleaseMenuButton/ReleaseMenuButton.tsx

```diff
@@ -1,72 +1,106 @@
-import {ArchiveIcon, EllipsisHorizontalIcon, TrashIcon, UnarchiveIcon} from '@sanity/icons'
+import {
+  ArchiveIcon,
+  CloseCircleIcon,
+  EllipsisHorizontalIcon,
+  TrashIcon,
+  UnarchiveIcon,
+} from '@sanity/icons'
 import {type DefinedTelemetryLog, useTelemetry} from '@sanity/telemetry/react'
 import {Menu, Spinner, Text, useToast} from '@sanity/ui'
 import {useCallback, useMemo, useState} from 'react'
 import {useRouter} from 'sanity/router'
 
 import {Button, Dialog, MenuButton, MenuItem} from '../../../../../ui-components'
 import {Translate, useTranslation} from '../../../../i18n'
-import {ArchivedRelease, DeletedRelease} from '../../../__telemetry__/releases.telemetry'
+import {
+  ArchivedRelease,
+  DeletedRelease,
+  UnscheduledRelease,
+} from '../../../__telemetry__/releases.telemetry'
 import {releasesLocaleNamespace} from '../../../i18n'
 import {type ReleaseDocument} from '../../../store/types'
 import {useReleaseOperations} from '../../../store/useReleaseOperations'
 import {getReleaseIdFromReleaseDocumentId} from '../../../util/getReleaseIdFromReleaseDocumentId'
 import {useBundleDocuments} from '../../detail/useBundleDocuments'
 
 export type ReleaseMenuButtonProps = {
-  disabled?: boolean
+  /** defaults to false
+   * set true if release primary CTA options should not
+   * be shown in the menu eg. unschedule, publish
+   */
+  ignoreCTA?: boolean
```

Comment:
Would it make more sense hideCTA? Even then not super satisfied with the name 🤔

Reply:
Agreed, not a fan of the name (that's why I felt the need to add a comment to make it clear). Perhaps `hideSchedulePublish` could work as those are the only 2 actions that'll need to be hidden